clients/cloud/kotlin/src/main/kotlin/io/confluent/examples/clients/cloud/ProducerExample.kt

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("ProducerExample")
+package io.confluent.examples.clients.cloud
+
+import io.confluent.examples.clients.cloud.model.DataRecord
+import io.confluent.examples.clients.cloud.util.loadConfig
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.clients.producer.RecordMetadata
+import org.apache.kafka.common.errors.TopicExistsException
+import org.apache.kafka.common.serialization.StringSerializer
+import java.util.*
+import java.util.concurrent.ExecutionException
+
+// Create topic in Confluent Cloud
+fun createTopic(topic: String,
+                partitions: Int,
+                replication: Int,
+                cloudConfig: Properties) {
+  val newTopic = NewTopic(topic, partitions, replication.toShort())
+
+
+  try {
+    val adminClient = AdminClient.create(cloudConfig)
+
+    adminClient.createTopics(listOf(newTopic)).all().get()
+
+  } catch (e: InterruptedException) {
+    // Ignore if TopicExistsException, which may be valid if topic exists
+    if (e.cause !is TopicExistsException) {
```

Comment:
As of me such complex exception handling here.
1. There are no checked exceptions in kotlin
2. Eventually we're catching everything but TopicExistsException.

It looks like we should do smth like

```
catch(e: Exception) {
  if (e.cause !is TopicExistsException) throw e
}
```